modules/appeals_api/app/services/appeals_api/central_mail_updater.rb

```diff
@@ -97,11 +91,8 @@ def parse_central_mail_response(raw_response)
 
     def update_appeal_status!(appeal, central_mail_status)
       attributes = CENTRAL_MAIL_STATUS_ATTRIBUTES.fetch(central_mail_status.status) do
-        log_message_to_sentry(
-          'Unknown status value from Central Mail API',
-          :warning,
-          status: central_mail_status.status
-        )
+        Rails.logger.warn('Unknown status value from EMMS API.' \
+                          " EMMS Status: #{central_mail_status.status}")
```

Comment:
I see the attempt to start using "EMMS API" rather than "Central Mail API" since the former is more accurate, but my point of view is that if only some areas of this file are renamed and not others, this creates more confusion. I'd rather we keep everything "Central Mail" and consider doing a full rename when we migrate Benefits Intake over to LHDI.

This looks like another opportunity for an approach like:
`Rails.logger.warn('Unknown status value from Central Mail API.', { central_mail_status: central_mail_status.status })`